pkg/registry/core/service/storage/storage.go

```diff
@@ -250,6 +266,33 @@ func (r *GenericREST) defaultOnReadService(service *api.Service) {
 	}
 }
 
+func (r *GenericREST) afterDelete(obj runtime.Object, options *metav1.DeleteOptions) {
+	svc := obj.(*api.Service)
+
+	klog.Errorf("TIM: afterDelete: svc %v/%v - prepare to deallocate", svc.Namespace, svc.Name)
```

Comment:
Commit `De-layer Service REST: Delete`:
it does not have a FIXME. Reminder to remove.